src/pages/registration/DescriptionPanel.tsx

```diff
@@ -61,11 +61,12 @@ const DescriptionPanel: FC<PanelProps> = ({ setTouchedFields }) => {
           {({ field, meta: { touched, error } }: FieldProps) => (
             <TextField
               {...field}
+              required
               data-testid="registration-title-field"
               inputProps={{ 'data-testid': 'registration-title-input' }}
               variant="outlined"
               fullWidth
-              label={t('common:title')}
+              label={<span>{t('common:title')}</span>}
```

Comment:
hvorfor ?

Reply:
ups, gammel moro... trodde jeg måtte gjøre noe lurt her for å legge til rød asterisk selv for hvert felt, før jeg fant ut jeg kunne sette required prop, og sette den til rød i theme